### PR TITLE
Add pass.html.erb and added Settings link to navbar

### PR DESCRIPTION
**Pull request by Logan-K**
Still needs confrim pass functionality but you can change your password if need be in its current state

### CODE REVIEW

**Comment by epugh** [comment]
So, is this access controlled?   Either a coach or you should be able to change the data.  I wonder if we need to introduce a helper like "can_access" or "has_permissions?" that does the "Am I a coach, or am I the current runner" logic?

**Comment by epugh** [comment]
because coaches need to be able to reset passwords as well.   Kate put one girl in, and then can't figure out the password cause there was a typo!

File: app/controllers/runners_controller.rb [hunk 1]
```diff
@@ -97,6 +97,10 @@ def today
   end
 
 
+  def pass
```
Comment (epugh) [inline app/controllers/runners_controller.rb]: Should this be "password", why shorten it?

**Comment by epugh** [comment]
Looks like good progress, and we need this!   Who can change a password?  Hopefully a coach can do it to "reset" a password, and you can change your own password.

**Comment by epugh** [comment]
@Logan-K Not sure if this is a ready to go PR?

**Comment by Logan-K** [comment]
Yes definitely not yet. Let me keep tackling this early this week here.

**Comment by epugh** [comment]
Is this ready to go?

**Comment by Logan-K** [comment]
Good to go ^. I ran through every combination I could think of for testing and I think everything is good. Wouldn't hurt to pull down real quick and kick the tires. If you don't run into any problems then it's good to merge

**Pull request merged**